src/components/ListUserSearch/ListUserSearch.js

```diff
@@ -48,4 +46,11 @@ export default function ListUserSearch(props) {
             ))}
         </Select>
     )
-}
\ No newline at end of file
+}
+
+const mapStateToProps = state => ({
+    usersApp: state.usersApp.data,
+    cards: state.cards.data
```

Comment:
it seems to me that every component that need for example `cards` from redux use `connect` to get it but it's wrong! you need to subscribe some parent and pass `cards` prop to children